integrations/spark/src/main/java/marquez/spark/agent/MarquezContext.java

```diff
@@ -33,14 +34,16 @@ public MarquezContext(ArgumentParser argument) throws URISyntaxException {
   public void emit(LineageEvent event) {
     try {
       // Todo: move to async client
+      log.debug("Posting LineageEvent {}", event);
       ResponseMessage resp = client.post(lineageURI, event);
+      ObjectMapper mapper = OpenLineageClient.createMapper();
```

Comment:
we wouldn't want to create a mapper pre request, lets just have it as a marquezcontext field.